Makefile

```diff
@@ -45,7 +47,13 @@ $(SUBDIRS):
 	$(MAKE) -C $@
 
 proto:
-	PATH=$(BINPATH):$(PATH) $(MAKE) -C proto/ proto
+	DOCKER_BUILDKIT=1 docker build \
+		--progress=plain \
```

Comment:
Do you really need `--progress plain` and `--quiet` here?

Reply:
Nope.